resources/home/dnanexus/utils/io.py

```
@@ -37,16 +46,46 @@
         session_file (string): IGV session file URL
+
+    Raises:
+        RuntimeError: Raised if invalid build param provided
     """
     order_map = set_order_map()
 
-    # Copy template to avoid overwriting
-    with open("/home/dnanexus/cnv-template.json") as fh:
+    with open(
+        "/home/dnanexus/cnv-template.json", encoding="utf-8", mode="r"
+    ) as fh:
```

Ruff (0.8.0)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)